src/web/components/SeriesSectionLink.tsx

```diff
@@ -77,16 +66,31 @@ const invertedStyle = css`
 	}
 `;
 
-const fontStyles = css`
-	font-weight: 700;
-	${headline.xxxsmall({ fontWeight: 'bold' })};
+const fontStyles = (format: Format) => css`
+	${format.theme === Special.Labs
+		? textSans.medium({ fontWeight: 'bold' })
+		: headline.xxxsmall({ fontWeight: 'bold' })}
 	${from.leftCol} {
-		${headline.xxsmall({ fontWeight: 'bold' })};
+		${format.theme === Special.Labs
+			? textSans.large({ fontWeight: 'bold' })
+			: headline.xxsmall({ fontWeight: 'bold' })}
 	}
+
+	${format.theme === Special.Labs && 'line-height: 23px;'}
+	${from.leftCol} {
+		${format.theme === Special.Labs && 'line-height: 20px;'}
+	}
```

Comment:
I think it might be easier to read if you regroup 
`from.leftCol`
and
`format.theme === Special.Labs`
instead of having them 2x as conditions

Reply:
Good idea 👍